synapse/cortex.py

```diff
@@ -3480,6 +3486,11 @@ async def _checkLayerModels(self):
         mrev = s_modelrev.ModelRev(self)
         await mrev.revCoreLayers()
 
+    @s_nexus.Pusher.onPushAuto('model:update')
+    async def updateModel(self, vers):
+        await self.cellinfo.set('cortex:model:version', vers)
+        self.model.vers = vers
```

Comment:
Lets make this a call back like `setModelVers()` on the `Model` object ( for reasons i mention below :D )